src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -160,314 +145,62 @@ export const RegistrationSearchBar = ({ registrationQuery }: Pick<SearchPageProp
-                    const seriesName = seriesLabels ? getLanguageString(seriesLabels) : '';
-                    if (seriesName) {
-                      fieldValueText = seriesName;
-                    } else {
-                      fieldValueText = (
-                        <SelectedSeriesFacetButton seriesIdentifier={typeof value === 'string' ? value : value[0]} />
-                      );
-                    }
-                    break;
-                  }
-                  case ResultParam.Journal: {
-                    fieldName = t('registration.resource_type.journal');
-                    const journalLabels = registrationQuery.data?.aggregations?.journal?.find(
-                      (bucket) => bucket.key === value
-                    )?.labels;
-                    const journalName = journalLabels ? getLanguageString(journalLabels) : '';
-                    if (journalName) {
-                      fieldValueText = journalName;
-                    } else {
-                      fieldValueText = (
-                        <SelectedJournalFacetButton journalIdentifier={typeof value === 'string' ? value : value[0]} />
-                      );
-                    }
-                    break;
-                  }
-                  case ResultParam.ScientificReportPeriodSinceParam: {
-                    fieldName = t('basic_data.nvi.nvi_publication_year');
-                    fieldValueText = value;
-                    break;
-                  }
-                  case ResultParam.Files: {
-                    fieldName = t('registration.files_and_license.files');
-                    fieldValueText = getFileFacetText(value as AggregationFileKeyType, t);
-                    break;
-                  }
-                  default:
-                    fieldValueText = typeof value === 'string' ? value : t('common.unknown');
-                }
-
-                if (!fieldName || !fieldValueText) {
-                  return null;
-                }
-
-                return (
-                  <Button
-                    key={`${param}-${value}`}
-                    data-testid={dataTestId.startPage.advancedSearch.removeFacetButton}
-                    variant="outlined"
-                    size="small"
-                    title={t('search.remove_filter')}
-                    sx={{ textTransform: 'none' }}
-                    endIcon={<ClearIcon />}
-                    onClick={() => {
-                      const syncedParams = syncParamsWithSearchFields(searchParams);
-                      const newParams = removeSearchParamValue(syncedParams, param, value);
-                      newParams.delete(ResultParam.From);
-                      navigate({ search: newParams.toString() });
-                    }}>
-                    {fieldName}: {fieldValueText}
-                  </Button>
-                );
-              })}
-            </Box>
+          {values.properties.length > 0 && (
```

Comment:
La til denne sjekken for å unngå tom div som gjør at det ble en veldig stor diff pga ny indentering 🥴